docs/resources/artifactory_local_conda_repository.md

```diff
@@ -0,0 +1,21 @@
+# Artifactory Local Conda Repository Resource
```

Comment:
So, you've added these docs but no functional code. Are you just filling in the blanks?

Reply:
conda is added to the list of generic repos. So it will be automatically generated.